cosmos/dbt/graph.py

```diff
@@ -190,6 +196,25 @@ def load(
         logger.info("Total nodes: %i", len(self.nodes))
         logger.info("Total filtered nodes: %i", len(self.nodes))
 
+    def load_via_dbt_ls_cache(self) -> bool:
+        """
+        Load dbt ls cache
+        """
+        logger.info(f"Trying to parse the dbt project using dbt ls cache {self.cache_identifier}...")
+        if settings.enable_cache and settings.experimental_cache:
+            dbt_ls_cache = Variable.get(self.cache_identifier, "")
```

Comment:
It might be better to query the db directly, so you bypass any configured secrets backend.

Reply:
@jedcunningham how do you advise us to do this?
Wouldn't there be a risk that with this, we'd create a larger coupling of Cosmos to Airflow that could be more sensitive to different versions of Airflow?

Reply:
Jed, since this has been long standing, I'll be merging as it is - and I can make a follow up PR to address after your feedback.